api/common/v1alpha1/entitytype.go

```diff
@@ -0,0 +1,11 @@
+package v1alpha1
+
+// EntityType is the type for all the entity types.
+type EntityType string
```

Comment:
`EntityType` looks a bit ambiguous. In the most contexts of Kong it stands for the type of an entity in Kong gateways (`services`, `rotues`, `plugins`, ...). And we are using `EntityType` in this way in many places including KIC, KGO and decK.
Since this is used for decide whether the entity is expected to be created and reconciled or only mirrored from Konnect,  can we give it a clearer and distinguishing name, like `EntitySource` or `EntityReconcileMode`?

Reply:
It makes sense. I've used `EntitySource` 👍🏻